examples/Getting Started with Semiconductor Module/Getting Started with Semiconductor Module.py

```diff
@@ -0,0 +1,215 @@
+""" Getting Started with Semiconductor Module example.
+
+    Example NI TSM Test Program using simulated NI-Digital and NI-DCPower instruments
+"""
+
+import random
+
+import nidcpower
+import nidigital
+from nitsm.codemoduleapi import (
+    SemiconductorModuleContext,
+    code_module,
+    Capability,
+    InstrumentTypeIdConstants,
+)
```

Comment:
This is fine but I think I like this syntax better:
```suggestion
import nitsm.codemoduleapi as tsm
```
Then later use it as 
```python
tsm.SemiconductorModuleContext
tsm.code_module
```
Either choice is fine